examples/Android/SnowboyAlexaDemo/src/ai/kitt/snowboy/audio/RecordingThread.java

```diff
@@ -116,7 +123,7 @@ private void record() {
             if (null != listener) {
                 listener.onAudioDataReceived(audioBuffer, audioBuffer.length);
```

Comment:
(see comment below first)

It's better that you implement a `onRefreshFile()` (or `onFileSizeExceeded()`) interface/function in `listener` to ask the listener to delete and create a new file every 50MB.